randovania/exporter/patch_data_factory.py

```diff
@@ -48,5 +50,10 @@ def __init__(
     def game_enum(self) -> RandovaniaGame:
         raise NotImplementedError
 
-    def create_data(self) -> dict:
+    def create_game_specific_data(self) -> dict:
         raise NotImplementedError
+
+    def create_data(self) -> dict:
+        game_data = self.create_game_specific_data()
+
+        return patch(game_data, self.patches.custom_patcher_data, False)
```

Comment:
Can you write a test please, that tests that this behaves as it should with custom extra data?